pkg/virt-controller/watch/vmi.go

```diff
@@ -226,20 +228,27 @@ func (c *VMIController) runWorker() {
 	}
 }
 
+var virtControllerTracerVMI *traceUtils.Tracer
+
 func (c *VMIController) Execute() bool {
 	key, quit := c.Queue.Get()
 	if quit {
 		return false
 	}
+
+	virtControllerTracerVMI = traceUtils.NewTrace(time.Second, "virt-controller VMI workqueue", trace.Field{Key: "Workqueue Key", Value: key})
+
 	defer c.Queue.Done(key)
```

Comment:
actually, I bet we can just use a defer here for the ```LogIfLong``` function. That way we never have to worry about whether it get's called or not. 

```defer virtControllerTracerVMI.Trace.LogIfLong(virtControllerTracerVMI.Threshold)```